app/assets/javascripts/central.js

```diff
@@ -2,16 +2,16 @@ var Project = require('models/project');
 var User = require('models/user');
 var TabNotification = require('mixins/tab_notification');
 
-var ProjectView = require('views/project_view');
-var ProjectSearchView = require('views/project_search_view');
+import ProjectView from './views/project_view';
+import ProjectSearchView from './views/project_search_view';
 var ProjectVelocityView = require('views/project_velocity_view');
 
 import TourController from 'controllers/tour/TourController';
 import { subscribeToProjectChanges } from './pusherSockets';
 
 require('./global_listeners');
 
-var Central = module.exports = {
+const Central = {
```

Comment:
I think we can make it simpler and just export the function instead of this object and call it `CentralController`, and then we just import use it without the `.start` at the routes JS file.